Procfile

```diff
@@ -1 +1 @@
-web: node app.js
```

Comment:
This will break the Heroku deployment, if you want to change it for yourself locally I can add support for a custom environment variable file.